QueueProcessor1/QueueProcessor1/MainWindow.xaml.cs

```diff
@@ -34,12 +35,56 @@ public MainWindow()
                 queuegrid.ColumnDefinitions.Add(new ColumnDefinition());
                 queuegrid.Children.Add(MyControl1);
 
-            }
+            }*/
         }
 
-        private void Button_Click(object sender, RoutedEventArgs e)
+        private void buttonCalc (object sender, RoutedEventArgs e)
         {
+            Proc p0 = new Proc();
```

Comment:
Not sure we will be able to use this?  We can discuss.

Reply:
There's not another way for me to create a process is there? It's either I create processes and hand them to you, or I give you the data in pieces